src/contexts/ShoppingCartContext/index.tsx

```
@@ -0,0 +1,132 @@
+'use client';
+
+import React, { createContext, useContext, useEffect, useMemo, useReducer } from "react";
```

ESLint

[error] 'useMemo' is defined but never used.

(@typescript-eslint/no-unused-vars)